README.md

```diff
@@ -1,3 +1,49 @@
 # Weather App
 
-In this project, I created a weather forecast site using the open map weather API.
\ No newline at end of file
+In this project, I created a weather forecast site using the open map weather API.
+I got comfortable consuming the API using Async/Await to work with promises using the `fetch` function.
+
+![screenshot](./screenshot.jpg)
+
+## Built With
+
+- HTML
+- JavaScript
+- Webpack
+- BootStrap
+- OpenMapAPI
+
+## Live Demo
+[Live Demo Link](https://acushlakoncept.github.io/weather-app/)
+
+
+## Getting Started
+
+To get a local copy of the repository please run the following commands on your terminal:
+
+```bash
+$ cd <folder>
```

Comment:
Do we really need this first line of instruction?

Reply:
removed!